### PR TITLE
FEATURE: TMA-1240 Read SDK and brick versions from files

### PR DESCRIPTION
**Pull request by sadam21**

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

File: spec/unit/version_spec.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2010-2018 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+require 'gooddata/version'
+
+describe 'GoodData::Versions' do
```
Comment (panjan) [inline spec/unit/version_spec.rb]: The double colon notation normally signifies a module but `GoodData::Versions` doesn't exist.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by panjan** [comment]
ok to merge

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/170/ : SUCCESS in 27s

**Pull request merged**